common/djangoapps/student/helpers.py

```diff
@@ -743,3 +743,12 @@ def does_user_profile_exist(user):
         return hasattr(user, 'profile')
     except (ProgrammingError, ObjectDoesNotExist):
         return False
+
+def user_has_passing_grade_in_course(user, enrollment):
+    """
+    Check to see if a user has passing grade for a course
+    """
+    course_grade = CourseGradeFactory().read(user, course=enrollment.course_overview, create_if_needed=False)
```

Comment:
Does this work if the course overview doesn't exist?

Reply:
I made a change here to protect against an empty overview. I'm not sure what's best practice in Python here actually, so if you have any advice, I'd appreciate it.